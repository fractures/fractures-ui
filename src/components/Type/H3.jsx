import PropTypes from 'prop-types'
import styled from 'styled-components'

const propTypes = {
	children: PropTypes.node.isRequired,
	className: PropTypes.string
}

const H3 = styled.h3`
	margin-bottom: 0;
	margin-top: 0;

	font-size: 1.5rem;
	line-height: 1.875rem;
`

H3.propTypes = propTypes

export default H3
