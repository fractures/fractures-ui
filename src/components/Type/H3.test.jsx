import H3 from "./H3"

describe("H3", () => {
	const tree = renderer.create(<H3>Extra header</H3>).toJSON()

	it("Matches snapshot", () => {
		expect(tree).toMatchSnapshot()
	})
})
