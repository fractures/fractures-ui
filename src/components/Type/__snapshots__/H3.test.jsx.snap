// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H3 Matches snapshot 1`] = `
<h3
  className="sc-bdVaJa tYuzJ"
>
  Extra header
</h3>
`;
